doc/conf.py

```diff
@@ -152,7 +152,7 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-# html_theme_options = {}
+html_theme_options = {"navigation_with_keys": True}
```

Comment:
Would setting this to False, i.e. the new default, also work? Or do we still see the warning then?

Reply:
just did that, let's see what fate brings us :grin:

Reply:
lemme add a note like you did on the Tool PR